px-map-behavior-vector-layer.es6.js

```diff
@@ -0,0 +1,147 @@
+(function() {
+  'use strict';
+
+  /****************************************************************************
+   * BEHAVIORS
+   ****************************************************************************/
+
+  /* Ensures the behavior namespace is created */
+  window.PxMapBehavior = (window.PxMapBehavior || {});
+
+  /**
+   *
+   * @polymerBehavior PxMapBehavior.VectorLayer
+   */
+  PxMapBehavior.VectorLayerImpl = {
+    properties: {
+      /**
+       * An object that will contain the geojson data to display:
+       *
+       * @type {Object}
+       */
+      data: {
+        type: Object,
+        observer: 'shouldUpdateInst',
+        reflectToAttribute: true
+      },
+
+      /**
+       * An object that can specify the style
+       *
+       * @type {Object}
+       */
+      featureStyle: {
+        type: Object,
+        value: null
```

Comment:
No need to explicitly set its default value to `null`. If you don't put an initial value, it'll be undefined until it is set and still be falsey.